docker/templates/Dockerfile.test-database.m4

```
@@ -6,7 +6,7 @@ ARG DEBIAN_FRONTEND=noninteractive
 # install_extensions.sh removes certain build dependencies that we need, so we
 # can't install everything here.
 # Note: curl is also a dependency of carton.
-RUN apt_install(`ca-certificates curl sudo')
+RUN apt_install(`ca-certificates curl sudo bzip2')
```

Just keep alphabetical ordering:

```suggestion
RUN apt_install(`bzip2 ca-certificates curl sudo')
```